### DISCUSSION TITLE
Upload proof of your academic status

### DISCUSSION THREAD

**Discussion by vchk123** [community/community#121044] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

On "Upload proof of your academic status" page, I uploaded picture and clicked "process my application", but it just refreshes the page.

**Comment by eliz-amst** [comment]
Did the application appear on a side of the page?

**Comment by itzzjb** [comment, marked as answer]
Yes, check whether you are uploading a .jpeg or a.png file. It only supports those two kinds. Otherwise the page refreshes.

**Reply by rgreen32** [reply]
i had to shrink the image dimensions to bypass this behavior
